src/background.js

```diff
@@ -7,3 +7,13 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
         chrome.action.setPopup({ tabId, popup: "popup_disabled.html" });
     };
 });
+
+chrome.runtime.onInstalled.addListener(function (object) {
+    let externalUrl = "https://github.com/ranashreyas/nbgitpuller-link-generator-webextension/blob/popout/usage.md";
```

Comment:
This link would need to be the berkeley-dsep-infra repo.

Reply:
Would not be possible to test technically until merged, but the link would be "https://github.com/berkeley-dsep-infra/nbgitpuller-link-generator-webextension/blob/main/usage.md".